src/Tribe/Tickets_View.php

```diff
@@ -57,18 +57,49 @@ public static function hook() {
 		return $myself;
 	}
 
+	/**
+	 * Tries to Flush the Rewrite rules
+	 *
+	 * @return void
+	 */
+	public function maybe_regenerate_rewrite_rules() {
+		// if they don't have any rewrite rules, do nothing
+		// Don't try to run stuff for non-logged users (too time consuming)
+		if ( ! is_array( $GLOBALS['wp_rewrite']->rules ) || ! is_user_logged_in() ) {
```

Comment:
The problem is that for users that update via FTP it doesn't work, creates support pain.